sdfObject/sdfobject-level.sdf.json

```diff
@@ -85,138 +85,135 @@
           "default": []
         },
         "StartUpCurrentLevel": {
-          "label": "StartUpCurrentLevel", 
-          "type": "number",
-          "minimum": 1,
-          "maximum": 254
-        },
-        "StartUpCurrentLevelSpecialValues": {
-          "label": "StartUpCurrentLevelSpecialValues",
-          "type": "number",
-          "enum": [
-            "MinimumDeviceValuePermitted",
-            "SetToPreviousValue"
-          ]
+          "label": "StartUpCurrentLevel",
+          "type": "integer", 
```

Comment:
sdfChoice currently does not allow to "factor out" data shape qualities such as type etc.
Should it?

Reply:
I believe we added this to the current schema